### PR TITLE
Upgrade websocket library https://horizenlabs.atlassian.net/browse/SDK-331

### PR DESCRIPTION
**Pull request by sgonta**
Signed-off-by: sgonta <sgonta@horizenlabs.io>

### CODE REVIEW

**Comment by sgonta** [comment]
Results after upgrade
image

File: sdk/src/test/scala/com/horizen/websocket/client/WebSocketServerPushEndpoint.scala [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.horizen.websocket.client
 
-import javax.websocket.server.ServerEndpoint
-import javax.websocket.{OnOpen, Session}
+import jakarta.websocket.server.ServerEndpoint
+import jakarta.websocket.{OnOpen, Session}
```
Comment (i-Alex) [inline sdk/src/test/scala/com/horizen/websocket/client/WebSocketServerPushEndpoint.scala:4]: Please check if there are any licence conflicts caused by using Jakarta EE

File: qa/README.md [hunk 2]
```diff
@@ -7,33 +7,56 @@ It is possible to test a SC node or nodes with or without real MC node connectio
 
 **Requirements**
 
-Install Python 3
-
-**Additional settings**
+1. Install Python 3 anf PIP
```
Comment (i-Alex) [inline qa/README.md:10]: typo `anf` -> `and`

File: qa/README.md [hunk 3]
```diff
@@ -7,33 +7,56 @@ It is possible to test a SC node or nodes with or without real MC node connectio
...
+1. Example for Linux:
 ```
-BITCOINCLI="/home/user/zen/zen-cli"
-BITCOIND="/home/user/zen/zend"
-SIDECHAIN_SDK="/home/user/Sidechains-SDK"
+sudo nano /etc/environment
 ```
+2. In this file after Path from the new line put Environment variables:
+```
+BITCOINCLI=/home/yourName/yourProjectDirectory/zen/zen-cli
```
Comment (i-Alex) [inline qa/README.md:27]: Probably `zen/zen-cli`-> `zen/src/zen-cli`, right?
same below

**Review by i-Alex — APPROVED** [review]

**Pull request merged**